src/components/molecules/OcBreadcrumb/OcBreadcrumb.spec.js

```diff
@@ -20,15 +20,49 @@ describe("OcBreadcrumb", () => {
     expect(wrapper.props().variation).toMatch("lead")
     expect(wrapper.classes()).toContain("oc-breadcrumb-lead")
     expect(wrapper).toMatchSnapshot()
-  }),
-    it("displays all items", () => {
-      const wrapper = shallowMount(Breadcrumb, {
-        propsData: {
-          items,
-        },
-      })
+  })
+  it("displays all items", () => {
+    const wrapper = shallowMount(Breadcrumb, {
+      propsData: {
+        items,
+      },
+    })
+
+    expect(wrapper.findAll(".oc-breadcrumb-list-item").length).toBe(4)
+    expect(wrapper.findAll("#oc-breadcrumb-contextmenu-trigger").length).toBe(0)
```

Comment:
Line is not needed because you have a dedicated test for this (`does not display context menu trigger when no slot given` some lines below). Don't test things multiple times. ;-)